### PR TITLE
MODORDERS-24: Add support for composite PO line

### PR DESCRIPTION
**Pull request by mreno-EBSCO**
The composite_purchase_order schema was updated to refer to a new composite_po_line schema that combines the po_line schema and all of its UUID references as schema "refs".

### CODE REVIEW

File: src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java [hunk 1]
```diff
@@ -125,23 +126,35 @@ public void testPoCreationFailure(TestContext ctx) throws Exception {
...
-      .header(X_OKAPI_TENANT)
-      .contentType(APPLICATION_JSON)
-      .body(body)
-      .post(rootPath)
-      .then()
-      .contentType(TEXT_PLAIN)
-      .statusCode(400)
-      .extract()
-      .response();
+    final Async asyncLocal = ctx.async();
```
Comment (craigmcnally) [inline src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java]: Is this needed?  If the rest assured call is synchronous (I think they are), then which part of this is asynchronous?

File: src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java [hunk 2]
```diff
@@ -125,23 +126,35 @@ public void testPoCreationFailure(TestContext ctx) throws Exception {
...
+        .header(X_OKAPI_TENANT)
+        .contentType(APPLICATION_JSON)
+        .body(body)
+      .when()
+        .post(rootPath)
+      .then()
+        .log()
+          .all()
+        .contentType(APPLICATION_JSON)
+        .statusCode(422)
```
Comment (craigmcnally) [inline src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java]: It's pretty cool that you get the 422/Unprocessible Entity for free... I was unaware of that and was confused when I saw this until I looked at the RMB code.

File: src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java (hunk 1 above)
Comment (mreno-EBSCO) [reply src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java]: Sorry, habit when dealing with vertx tests. :) Agreed, while it doesn't hurt, in this case it is not needed. I'll remove it.

File: src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java (hunk 2 above)
Comment (mreno-EBSCO) [reply src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java]: Yeah, pretty much anything that causes schema validation errors is reported as a 422 by Okapi. Sorry for the confusion.

**Pull request merged**